locales/ara.json

```diff
@@ -149,8 +151,8 @@
     "qstnMarkToolTip": "معرف التطبيق متاح في الإقرار الذي تم استلامه بعد التسجيل."
   },
   "IdInputModal": {
-    "header": "أدخل UIN / VID لتنزيل {{vcLabel}}",
```

Comment:
As Mentioned in the PR description UIN / VID should also be translated, but the translations are missing here. 
1. Either UIN / VID has to be translated post confirming with Pragya. 
2. or update the PR Description.

Reply:
> As Mentioned in the PR description UIN / VID should also be translated, but the translations are missing here.
> 
> 1. Either UIN / VID has to be translated post confirming with Pragya.
> 2. or update the PR Description.

Apologies for the wrong update in PR description, as a part of this card only the constants like Card, Verifier, Wallet, Bluetooth, Selfie will be translated. Have updated PR description with the same